packages/components/src/Card/Card.spec.js

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Card from './Card';
+
+describe('@components/card', () => {
+  it('Given a normal call It should Render', () => {
```

Comment:
Lets add more test, for example, that if you add a Header it is renderd, same for children, and lets test that de paper has padding in 0.